.expeditor/scripts/verify/shared.sh

```diff
@@ -17,17 +20,7 @@ get_rustfmt_toolchain() {
   # break the way rustfmt uses rustc. Therefore, before updating the pin below, double check
   # that the nightly version you're going to update it to includes rustfmt. You can do that
   # using https://mexus.github.io/rustup-components-history/x86_64-unknown-linux-gnu.html
-  cat "$dir/../../RUSTFMT_VERSION"
-}
-
-# Get the version of the nightly toolchain we use for compiling,
-# running, tests, etc.
-get_nightly_toolchain() {
-    cat "$dir/../../RUST_NIGHTLY_VERSION"
-}
-
-get_toolchain() {
-  cat "$dir/../../rust-toolchain"
+  cat "RUSTFMT_VERSION"
```

Comment:
I think it would be more robust to use the relative pathing here (like `get_toolchain` in `.expeditor/scripts/shared.sh`)

Reply:
I'll  fix this